raiden/accounts.py

```diff
@@ -89,7 +89,7 @@ def __init__(self, keystore_path=None):
                 files = os.listdir(self.keystore_path)
             except OSError as ex:
                 msg = 'Unable to list the specified directory'
-                log.error('%s %s: %s', msg, self.keystore_path, ex)
+                log.error('OsError', msg=msg, path=self.keystore_path, ex=ex)
```

Comment:
```python
import structlog
def f(): g()
try:
    f()
except Exception as e:
    structlog.getLogger().exception('blah', e=e)
```
This test script will print:

> 2018-05-22 17:26.32 blah                           e=NameError("name 'g' is not defined",)

Can we get the stack trace back? Having a structured logging is cool, but exception logging without stack traces is useless.

Reply:
@hackaugusto 

it works like this

```
import structlog

log = structlog.get_logger()
def f(): g()
try:
    f()
except Exception as e:
    log.error('HALP', e=e, exc_info=True)
```

Of course it might be possible to configure `structlog` to include traceback automatically for `exception` log level.

Reply:
In stdlib logging `exception` is just an alias for `error` that adds `exc_info=True`.